src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -534,6 +534,33 @@ def prepare_latents(self, batch_size, num_channels_latents, height, width, dtype
         latents = latents * self.scheduler.init_noise_sigma
         return latents
 
+    def enable_freeu(self, s1=0.9, s2=0.2, b1=1.2, b2=1.4):
```

Comment:
Can we add some more in-detail docstring here explaining what each variable does? Also it'd be very nice to have some more expressive naming

Reply:
PR is not ready for a first review. You will be notified once that is the case.

Reply:
Tried explaining.

Reply:
IDK how I feel about hard-coding the defaults based on one specific model. If people just do `enable_freeu` they end up getting the v1-5 params regardless of the unet they are using, no?

Would it make sense for us to not have any defaults, but maybe create a util that contains the good FreeU values for each base model that if the person imports that util it auto-selects. It could also be a tiny separate pypi package 🤔

Reply:
No need for a tiny package IMO. We can have no defaults and maybe suggest values that reasonably work well in the documentation.

Reply:
Yeah having it in the docs is nice, but I think having utils/tiny package that you do `enable_freeu` and it just works has its power on the DX imo, but I understand it may be hard to maintain/tricky to add, so not a strong opinion, unless we find a cool way to do this cc @patrickvonplaten for ideas too if any

Reply:
Removed defaults.